amethyst_ui/src/event.rs

```diff
@@ -48,10 +48,15 @@ pub enum UiEventType {
         /// The entity on which the dragged object was dropped.
         dropped_on: Entity,
     },
-    /// When the value of an `Editable` Ui element has changed.
-    Change,
-    /// When an `Editable` Ui element has been committed.
-    Commit,
+    /// When the value of a UiText element has been changed by user input.
+    ValueChange,
+    /// When the value of a UiText element has been committed either by 
+    /// losing focus on mobile or by pressing "Enter".
+    ValueCommit,
+    /// When an editable UiText element has gained focus.
+    Focus,
```

Comment:
what about GainedFocus + LostFocus? :)

Reply:
I just lifted the names of the DOM events so they'd be familiar (at least to those familiar with web development).